doc/HOWTO_RELEASE.rst.txt

```diff
@@ -182,7 +182,7 @@ After a date is set, create a new maintenance/x.y.z branch, add new empty
 release notes for the next version in the master branch and update the Trac
 Milestones.
 
-Make sure current trunk builds a package correctly
+Make sure current branch builds a package correctly
 --------------------------------------------------
```

Comment:
This should say `master` or `master branch` not `branch.

Also, you need to make the length of the `----` match this heading, else we'll get warnings.

Reply:
Alright, I'll take care of this from now and will make the changes.
Besides, regarding the wording change in the above line, I was actually a bit unsure about it. Since we need to create a new maintenance branch as mentioned in the previous para, and the current line also says it to be the ```current trunk```, I felt that it is mentioning about the ```current maintenance branch``` only and not the ```master``` branch and hence needed your views.

Reply:
Might have missed something there, will reread tomorrow.

Reply:
merging this as-is, if further changes are needed we can do them in a subsequent PR